src/core/federation.py

```diff
@@ -452,16 +452,16 @@ async def get_project_epds(project_id: str, token: str) -> list[dict]:
     return data.get("projectEpds")
 
 
-async def delete_assembly(id: str, token: str) -> dict:
+async def delete_assemblies(ids: list[str], token: str) -> dict:
     """Delete assembly"""
```

Comment:
Tiny thing here; the comment should be updated to match the new functionality :)